src/lib.rs

```diff
@@ -7,8 +7,10 @@
 /// Compliance
 mod common;
 
+use crate::configuration::Configuration;
```

Comment:
This should be at line :23 (rustfmt will collapse it with the "validate_country_code" include.